packages/html_shim/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Shim that exposes dart:html in a Flutter application throwing excep
 author: Dart Team <misc@dartlang.org>
 homepage: https://github.com/flutter/devtools/tree/master/packages/html_shim
 
-version: 0.0.1
+version: 0.0.2
 
 environment:
   sdk: '>=2.0.0 <3.0.0'
```

Comment:
Should we lock the sdk to 2.5.0?

Reply:
hmm. If there are ever any problems with this package not working with newer environments we can do that. my expectation is this package will continue to work fine with new sdk rolls assuming we aren't changing the public apis for dart:html and dart:js in breaking ways.